.github/workflows/icons.yml

```diff
@@ -0,0 +1,59 @@
+name: Download icons from Figma
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+
+jobs:
+  download:
+    runs-on: ubuntu-latest
+    name: Download icons from Figma
+    defaults:
+      run:
+        working-directory: ./packages/icons
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          check-latest: true
+          registry-url: "https://registry.npmjs.org/"
+          scope: "@talend"
+          cache: "yarn"
+
+      - name: Download icons
+        run: npx @talend/figma-icons-downloader
```

Comment:
where is the code of @talend/figma-icons-downloader ?

Reply:
https://github.com/Talend/ux-tools :)